source/feature/remove-snapshot.html.md

```diff
@@ -0,0 +1,116 @@
+
+# Remove Snapshot
```

Comment:
Missing the headers like for example :
title: DiscardAfterDelete
category: feature
authors: ishaby
feature_name: Discard after delete
feature_modules: vdsm
feature_status: Partially implemented in oVirt 4.0